open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventConsumer.java

```diff
@@ -171,14 +187,7 @@ public void run()
             }
             finally
             {
-                try
-                {
-                    Thread.sleep(1000);

```

Comment:
Why are we delaying here again? This will slow down the polling thread.

Reply:
We are delaying here so that we only run this check or do a poll every 1 second.  I know it is odd to have the sleep in 2 different places, but only one of them actually is used.